inst/include/dust/random/utils.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef DUST_RANDOM_UTILS_HPP
+#define DUST_RANDOM_UTILS_HPP
+
+#include <dust/random/numeric.hpp>
+
+namespace dust {
+namespace random {
+
+enum xoshiro_mode {STARSTAR, PLUSPLUS, PLUS};
+
+template <typename T, typename U>
+T int_to_real(U value);
```

Comment:
Is this prototype/template needed, as all possibilities are explicitly defined?

Reply:
yes, because non-templated functions can't vary only in their return value